Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -196,10 +195,10 @@ internal static Task ApplyDrawableAsync(this IShellContext shellContext, Bindabl
 								return;
 
 							// we are back, so update the working element
-							element = bindable ?? renderer.Element;
+							element = bindable ?? renderer?.Element;
 
 							// makse sure things are good now that we are back
-							if (element == null || renderer.View == null)
+							if (element == null || (renderer != null && renderer.View == null))
```

Comment:
In line 176
```C#
if (element == null || renderer?.View == null)
```
I think it's worth doing the same in 2 places.

Reply:
The reason I didn't do it with the operator is that I don't want a null renderer to return here.

If the renderer is null then this logic doesn't apply. This logic only applies if there is a renderer being passed in

Reply:
But that does mean I should apply the same thing at line 176 :-p